langchain/chains/api/prompt.py

```diff
@@ -2,12 +2,13 @@
 from langchain.prompts.prompt import PromptTemplate
 
 API_URL_PROMPT_TEMPLATE = """You are given the below API Documentation:
-
 {api_docs}
+Using this documentation, generate the full API url to call for answering the user question.
+You should build the API url so as to get a response that is as short as possible, while still getting the necessary information to answer the question. Pay attention in the following example, how there are many pieces of information that are deliberately not included in the API call. You should do the same.
```

Comment:
Given that we have no example, I would change this for:

"You should build the API url so as to get a response that is as short as possible, while still getting the necessary information to answer the question. Pay attention to deliberately exclude unnecessary pieces of information in the API call."